rusoto/services/sts/src/lib.rs

```diff
@@ -1,7 +1,45 @@
 
 //! AWS Security Token Service
 //!
-//! If you're using the service, you're probably looking for [StsClient](struct.StsClient.html) and [Sts](trait.Sts.html).
+//! If you're using the service, you're probably looking for [StsClient](struct.StsClient.html), [Sts](trait.Sts.html)
+//! and [StsAssumeRoleSessionCredentialsProvider](struct.StsAssumeRoleSessionCredentialsProvider.html).
```

Comment:
Good catch, didn't notice there's a template for that. https://github.com/rusoto/rusoto/blob/master/service_crategen/src/main.rs#L259-L270. I'm going to update my patch :)